bifacial_radiance/modelchain.py

```diff
@@ -184,204 +117,66 @@ def runModelChain(simulationParamsDict, sceneParamsDict, timeControlParamsDict=N
-                    # trackerdict[time]['scene'] = scene since we don't do makeScene1axis
-                    for i in trackerdict:
-                        trackerdict[i]['scene'] = scene
-                        trackerdict[i]['radfile'] = scene.radfiles
-                    return trackerdict
-                
-                trackerdict = _addRadfile(trackerdict) # instead of makeScene1axis
-
-                footime=0
-                for time in sorted(timelist):  
-                    footime = footime+1
-                    trackerdict = demo.makeOct1axis(trackerdict, singleindex=time,
-                                                    hpc=simulationParamsDict['hpc'])
-                    trackerdict = demo.analysis1axis(trackerdict, singleindex=time,
-                                                     modWanted=analysisParamsDict['modWanted'],
-                                                     rowWanted=analysisParamsDict['rowWanted'],
-                                                     sensorsy=analysisParamsDict['sensorsy'])
-                    analysis = trackerdict[time]['AnalysisObj']  # save and return the last run
-                    print('Bifacial ratio average for %d out of %d datapoints:  %0.3f' % ( footime,
-                                    timelist.__len__(), 
-                                    np.sum(demo.Wm2Back) / np.sum(demo.Wm2Front)))
-
-    else:  # Tracking
-        print('\n***Starting 1-axis tracking simulation***\n')
-        
-        if simulationParamsDict['timeIndexSimulation']:
-            raise Exception('timeIndexSimulations not currently '+
-                            'supported for tracking')
-    
-        if 'gcr' not in sceneParamsDict:  # didn't get gcr passed - need to calculate it
-            sceneParamsDict['gcr'] = moduleDict['sceney'] / \
-                sceneParamsDict['pitch']
-        trackerdict = demo.set1axis(metdata, axis_azimuth=sceneParamsDict['axis_azimuth'],
-                                    gcr=sceneParamsDict['gcr'],
-                                    limit_angle=trackingParamsDict['limit_angle'],
-                                    angledelta=trackingParamsDict['angle_delta'],
-                                    backtrack=trackingParamsDict['backtrack'],
-                                    cumulativesky=simulationParamsDict["cumulativeSky"])
-
-        if simulationParamsDict["cumulativeSky"]:  # cumulative sky routine
+    else:
+    # Run everything through TrackerDict.    
+
+        if simulationParamsDict['tracking'] == False:
+            trackerdict = demo.set1axis(metdata, 
+                                         cumulativesky=simulationParamsDict["cumulativeSky"],
+                                        fixed_tilt_angle=sceneParamsDict['tilt'],
+                                        fixed_tilt_azimuth=sceneParamsDict['azimuth'])
+        else:
+            trackerdict = demo.set1axis(metdata, gcr=sceneParamsDict['gcr'],
```

Comment:
parameter `axis_angle` is not passed anywhere here...

Reply:
What do you mean by axis_angle? axis_tilt? It is not an input in the outside set1axis functions anyways, I made a note to correct that as it is in the internal _set1axis

Reply:
We also don't really support it, I think it prints a warning, as we don't tilt the scene generation and/or the sensor positions.